src/olympia/devhub/tests/test_views.py

```diff
@@ -57,6 +57,11 @@ def setUp(self):
         assert self.client.login(email='regular@mozilla.com')
         assert self.client.get(self.url).status_code == 200
         self.user_profile = UserProfile.objects.get(id=999)
+        not_their_addon = addon_factory(users=[user_factory()])
```

Comment:
Nit: It actually *is* their addon, isn't it? It's just that the `AddonUser` record has them listed as deleted.

Reply:
It's not their add-on _any more_  😉 - they've been deleted from the addon owners.  (A user would never be added as deleted in reality)